dali/pipeline/operators/decoder/nvjpeg_decoder_decoupled_api.h

```diff
@@ -232,19 +232,20 @@ class nvJPEGDecoder : public Operator<MixedBackend>, CachedDecoderImpl {
       const auto file_name = in.GetSourceInfo();
       cudaStream_t stream = ws->stream();
       auto *output_data = output.mutable_tensor<uint8_t>(i);
```

Comment:
the code here seems to be much simpler than the single operator version. Can you do the same thing in the single operator, for the sake of consistency?

Reply:
Done.